ide-intellij.html

```diff
@@ -0,0 +1,222 @@
+    <kbd>mainClass</kbd> accordingly to <kbd>org.openjfx.MainApp</kbd>.
+</p>
+<p>
+    Add the required dependencies for <kbd>javafx.controls</kbd> and <kbd>javafx.fxml</kbd>. 
+    <a href="images/ide/intellij/maven/idea03.png" target="_blank"><img src="images/ide/intellij/maven/idea03.png" alt="Update the pom"/></a>
+   
+    You can download the resulting pom file from <a href="images/ide/intellij/maven/pom.xml">here</a>.
+</p>
+
+<p>
+    Import the changes. The JavaFX classes will be recognized. Notice also that Maven manages 
+    the required dependencies: it will add <kbd>javafx.base</kbd> and 
+    <kbd>javafx.graphics</kbd> that are required by <kbd>javafx.controls</kbd> and <kbd>javafx.fxml</kbd>, 
+    but most important, it will add the required classifier based on your platform, downloading 
+    the specific platform jars.
+    <a href="images/ide/intellij/maven/idea04.png" target="_blank"><img src="images/ide/intellij/maven/idea04.png" alt="Platform dependencies"/></a>
+   
+    As for any other maven dependencies, these jars can be found in the local <kbd>.m2</kbd> repository.
+</p>
+
+<p>
+    <strong>3. Run the project</strong>
+</p>
+<p>
+    You can open the Maven Projects window and click on <kbd>HelloFX->Plugins->compiler->compiler:compile</kbd> to 
+    compile the project, and on <kbd>HelloFX->Plugins->exec->exec:java</kbd> to execute the project.
+    <a href="images/ide/intellij/maven/idea05.png" target="_blank"><img src="images/ide/intellij/maven/idea05.png" alt="Run project"/></a>
+</p>
+<p>
+    You can also open a terminal and run <kbd>mvn compile exec:java</kbd> to run the project.
+</p>
+
+<a name="Gradle"></a><h3>Gradle</h3>
+
+<p>
+    Follow these steps to create a JavaFX project and use the Gradle tools to build it and run it.
+</p>
+
+<p>
+    <strong>1. Create a Gradle project</strong>
+</p>
+<p>
+    Create a Gradle project with Java.
+    <a href="images/ide/intellij/gradle/idea01.png" target="_blank"><img src="images/ide/intellij/gradle/idea01.png" alt="Create a Gradle project"/></a>
+   
+    Provide the groupId, like <kbd>org.openjfx</kbd>, the artifactId, like <kbd>hellofx</kbd>. 
+    Select the Gradle JVM based on the project JDK <span class="JDK">11</span>.
+    Then provide a name to the project, like <kbd>HelloFX</kbd> and a location for the project.
+    
+    When the project opens, add a package and an empty <kbd>HelloFX</kbd> class.
+    
+    <a href="images/ide/intellij/gradle/idea02.png" target="_blank"><img src="images/ide/intellij/gradle/idea02.png" alt="Open project"/></a>
+</p>
+
+<p>
+    <strong>2. Modify the build</strong>
+</p>
+<p>
+    Edit the <kbd>build.gradle</kbd> file and replace it with this <a href="build.gradle">build</a> file, setting the 
+    <kbd>mainClassName</kbd> accordingly to <kbd>hellofx.HelloFX</kbd>.
+</p>
+<p>
+    Note we have added the required dependencies with the classifier for the platform. 
```

Comment:
Do we need this sentence? If yes, can we re-write it? Something like:

> Unlike Maven, platform needs to be manually determined and added as classifier to the dependencies.

Reply:
done